doc/cli/operator-sdk_scorecard.md

```diff
@@ -18,10 +18,10 @@ operator-sdk scorecard [flags]
       --config string       config file (default is '<project_dir>/.osdk-scorecard'; the config file's extension and format can be .yaml, .json, or .toml)
   -h, --help                help for scorecard
       --kubeconfig string   Path to kubeconfig of custom resource created in cluster
-  -L, --list                If true, only print the test names that would be run based on selector filtering (only valid when version is v1alpha2)
+  -L, --list                If true, only print the test names that would be run based on selector filtering
   -o, --output string       Output format for results. Valid values: text, json (default "text")
-  -l, --selector string     selector (label query) to filter tests on (only valid when version is v1alpha2)
-      --version string      scorecard version. Valid values: v1alpha1, v1alpha2 (default "v1alpha2")
+  -l, --selector string     selector (label query) to filter tests on
+      --version string      scorecard version. Valid values: v1alpha2 (default "v1alpha2")
```

Comment:
Should we change this flag to be hidden since there's only one valid value?

Reply:
I'd like to think about this one and make this decision in the next PR if possible.